evap/staff/importers.py

```diff
@@ -100,9 +104,7 @@ def store_in_database(self, vote_start_datetime, vote_end_date, semester):
             course=course,
         )
         evaluation.save()
-        # This is safe because the user's email address is checked before in the importer (see #953)
-        responsible_dbobj = UserProfile.objects.get(email=self.responsible_email)
-        evaluation.contributions.create(contributor=responsible_dbobj, evaluation=evaluation, responsible=True, can_edit=True, textanswer_visibility=Contribution.GENERAL_TEXTANSWERS)
+        evaluation.contributions.create(contributor=responsible_dbobj, evaluation=evaluation, can_edit=True, textanswer_visibility=Contribution.GENERAL_TEXTANSWERS)
```

Comment:
Same as for the single results here - we are unconditionally generating a contribution where the responsible is the contributor. Why?

Reply:
here it makes sense - we assume the responsible person is also contributing to the course, so we can add a contribution during import. in case these people do not actually contribute and shall not be part of the evaluation, they will be removed later by managers or editors. but that's not usually the case.